src/briefcase/windows.py

```diff
@@ -123,10 +126,9 @@ def walk_dir(path, depth=0):
                 for entry in entries:
                     exe_name = entry.split('=')[0].strip()
                     description = self.distribution.get_description()
-                    shortcutid = uuid.uuid4().hex
                     shortcuts.append("""\
                             <Shortcut
-                                Id="AppShortcut_{shortcutid}"
+                                Id="Shortcut_%s"
```

Comment:
It is mixing old and new, that's true. IIRC I did this so it could be parsed separately, the initial template fill handles the `.format()` args, but that shortcut name needs to be provided later once the id is generated.
Looking over the code again now, I suspect this is a holdover from an earlier implementation of it where the id was matching something that wasn't known until later.... currently it's a uuid3 which could in all likelihood be replaces by any old uuid up on this line instead

Reply:
@andrewleech If this is a holdover from an earlier implementation, is this something we could fix to just directly populate a uuid3?

Reply:
I had a bit more of a look yesterday, my last commit on this branch was explicitly replacing a uuid setting at the top with what's the now, I can't remember exactly why it was needed though. I'll need to test it without that change again.

Reply:
Ah yes, the AppShortcut template generated is later used twice for both the application folder and the start menu shortcuts, this results in a duplicated Shortcut Id which Wix fails at.

Reply:
The shortcuts are being handled by re-writing the Wix config file, looking for those magic comment markers - would it be possible to roll that into the template itself? If we pass in extra context to the template, that block could be handled as a conditional (and/or loop) in the template, rather than needing a rewrite

Reply:
The problem with inclusion in the template is it needs to be duplicated for each entry-point defined in setup.py
All the other <File> and <Compoenent> sections are generated in a loop and injected into the markers in the same way, the shortcut block is generated as multiline for readability but it's really not that different.

Does cookiecutter support looping sections of a template? If so arguably all the content, contentrefs and shortcuts could be pushed to that perhaps?